src/native_module/core.cljs

```diff
@@ -14,51 +16,69 @@
 
 (defn account-manager
   []
-  (when (exists? (.-NativeModules react-native))
-    (.-AccountManager ^js (.-NativeModules react-native))))
+  (if false
+    (when (exists? (.-NativeModules react-native))
+      (.-AccountManager ^js (.-NativeModules react-native)))
+    status-backend/fetch-js-obj))
```

Comment:
along this file I use `if false` as a  temporary pattern to ignore the original module implementation and use the back-end fetch callback